### ISSUE TITLE
1.41.1 on Mojave washed out colors

### ISSUE THREAD

**Issue by dhowden** [microsoft/vscode#89576]
@bpasero  @deepak1556 Follow up to https://github.com/microsoft/vscode/issues/86849 (but this issue was automatically locked a few days ago).

Upgraded to 1.41.1 and noticed that the colors had dimmed again.

1.41.1

<img width="1136" alt="Screen Shot 2020-01-29 at 9 50 03 am" src="https://user-images.githubusercontent.com/2822/73315558-7ee0cf00-4284-11ea-8c21-7616e997e868.png">

1.41.0:

<img width="1136" alt="Screen Shot 2020-01-29 at 10 49 47 am" src="https://user-images.githubusercontent.com/2822/73315865-5e654480-4285-11ea-80fb-77d768939016.png">

I still have `"disable-color-correct-rendering": true` set (which was the solution to #86849).

**Comment by dhowden** [comment]
Ok, just tried the current Insiders build for 1.42.0 and it appears to be fixed (back to 1.41.0 colours again).

```
Version: 1.42.0-insider
Commit: be9c72410acc677070685a616773b122f4c5e141
Date: 2020-01-29T05:35:51.712Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 18.7.0
```

**Comment by dhowden** [comment]
Updated to 1.42.0 and this is fine, so closing.
